web/adelacecilia/html/data list/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<head>
+  <title> Data List </title>
+</head>
+  <body>
+    <label for="ice-cream-choice">
+      <h2> What's your favorite ice-cream? </h2>
+      <h3> Please select an option: </h3>
+    <input list="ice-cream-flavors"
+    id="ice-cream-choice"
+    name="ice-cream-choice" 
+    </label>
+
+    <datalist id="ice-cream-flavors">
```

Comment:
Please dont leave empty lines in your HTML :)

Reply:
oops...I did it again.  :[

Sorted!